ci/ray_ci/doc/cmd_build.py

```diff
@@ -14,11 +20,19 @@ def main(ray_checkout_dir: str) -> None:
     """
     This script builds ray doc and upload build artifacts to S3.
     """
+    ci_init()
+
     logger.info("Building ray doc.")
     _build(ray_checkout_dir)
 
+    if os.environ.get("BUILDKITE_BRANCH") != "master":
```

Comment:
should we check based on pipeline ID instead?

Reply:
not really because we only want to store caches for the master branch